src/core/Notification/Notification.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import {
+  HtmlDiv,
+  HtmlDivWithRef,
+  HtmlButton,
+  HtmlButtonProps,
+  HtmlDivWithRefProps,
+} from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+
+import { AutoId } from '../utils/AutoId/AutoId';
+import { getConditionalAriaProp } from '../../utils/aria';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { baseStyles } from './Notification.baseStyles';
+
+export const baseClassName = 'fi-notification';
+
+export const notificationClassNames = {
+  styleWrapper: `${baseClassName}_style-wrapper`,
+  content: `${baseClassName}_content`,
+  label: `${baseClassName}_label`,
+  textContentWrapper: `${baseClassName}_text-content-wrapper`,
+  icon: `${baseClassName}_icon`,
+  iconWrapper: `${baseClassName}_icon-wrapper`,
+  closeButton: `${baseClassName}_close-button`,
+  closeButtonWrapper: `${baseClassName}_close-button-wrapper`,
+  smallScreen: `${baseClassName}--small-screen`,
+  actionElementWrapper: `${baseClassName}_action-element-wrapper`,
+};
+
+export interface NotificationProps extends HtmlDivWithRefProps {
+  /** Style variant. Affects color and icon used.
+   * @default 'neutral'
+   */
+  status?: 'neutral' | 'success' | 'error';
```

Comment:
I think we discussed that the `success` variant is removed as it will be used in new component, **Toast**.

Reply:
Fixed. Removed success.